README.md

```diff
@@ -10,6 +10,7 @@ Stamp catalog items displayed in a list.
 Stamp data fetched from an API and saved in the app.
 Users can choose an item from the list and add it to the shopping cart.
 Upon checkout, the total amount to pay will be calculated.
+Added a new Onboarding Screen View which has a stamp list with a scroll view.
```

Comment:
Just a note, you want to try and leave changes related to your homework assignment in a pull request. I want to be able to leave feedback on your code, and a pull request conversation is a great way to do it!

Reply:
Hi David, thank you

I will make it to office hours next Monday.

With week04 assignment, I was coming up with ways to present the list using ForEach and was just testing different ways to present the view.  I should have placed the discountTypes array in the model struct but did not go back and re-work that portion.  Am still trying to organize how my app should be organized and this was a way to come up with different means.


As far as the pull request, I was having a difficult time with the errors I kept receiving and had to create a new repo.  I also kept forgetting to create a branch before one starts coding up.  I do agree with you 100% and will do better.